coursier.bzl

```diff
@@ -469,11 +469,14 @@ def _pinned_coursier_fetch_impl(repository_ctx):
         "load(\"@bazel_tools//tools/build_defs/repo:utils.bzl\", \"maybe\")",
         "def pinned_maven_install():",
     ]
+    maven_artifacts = []
     netrc_entries = {}
 
     for artifact in dep_tree["dependencies"]:
         if artifact.get("url") != None:
             http_file_repository_name = escape(artifact["coord"])
+            if not get_classifier(artifact["coord"]) == "sources":
```

Comment:
This check seems awfully fragile. I think it'll allow javadoc jars and platform binaries to be included, which probably isn't your intent. Perhaps an allow list of classifiers would be a better approach?

Reply:
Yes, maybe it's be better to expose more information (classifier + label?) and let consumers decide what to do with it, will look into that.

Reply:
I think you also want to reject javadoc jars as well since `rules_jvm_external` makes it easy to download those too.